files/bin/gc_builders

```diff
@@ -2,8 +2,7 @@
 
 echo "Removing containers $(date)"
 
-# Find builder containers at least a week old and remove them
-docker ps -a -f "name=builder-init-*" | awk '/((7|1[0-9]) days)|week|month/ { print $1 }' | xargs docker rm || true
-docker ps -a -f "name=builder-cloner-*" | awk '/((7|1[0-9]) days)|week|month/ { print $1 }' | xargs docker rm || true
-docker ps -a -f "label=com.codeclimate.role=builder" | awk '/((7|1[0-9]) days)|week|month/ { print $1 }' | xargs docker rm || true
-docker ps -a -f "name=cc-engines-*" | awk '/((7|1[0-9]) days)|week|month/ { print $1 }' | xargs docker rm || true
+for filter in "name=builder-init-*" "name=builder-cloner-*" "label=com.codeclimate.role=builder" "name=cc-engines-*"
+do
+  docker ps -a -f "$filter" | awk '/((7|1[0-9]) days)|week|month/ { print $1 }' | xargs docker rm || true
```

Comment:
Does this run in a linux container? If so, you might benefit from GNU `xarg`'s `--no-run-if-empty` option. Would be quieter and you could remove the potentially real-problem-masking `|| true`